Package.swift

```diff
@@ -33,8 +33,8 @@ let package = Package(
         .executable(name: "SwiftMetricsCommonSample", targets: ["SwiftMetricsCommonSample"]),
     ],
   dependencies: [
-    .package(url: "https://github.com/IBM-Swift/Kitura.git", from: "2.1.0"),
-    .package(url: "https://github.com/IBM-Swift/Kitura-WebSocket.git", from: "1.0.0"),
+    .package(url: "https://github.com/IBM-Swift/Kitura.git", .branch("master")),
```

Comment:
This is just to keep CI passing, before merge it should be changed to Kitura 2.3 (which doesn't yet exist)